modules/_node-scripts/tsconfig/writeProjectTsconfig.mjs

```diff
@@ -90,11 +92,26 @@ export default async function writeProjectTsconfig(
 		references,
 	};
 
+	json['@generated'] = hash(json);
+
 	sortObjectKeys(json);
 
 	await fs.writeFile(
 		path.join(srcPath, 'tsconfig.json'),
-		JSON.stringify(json, null, '\t'),
+		stringifyJson(json),
```

Comment:
I created a `sortObjectKeys()` function in the top level `util` folder to make SF happy. Maybe it is not needed and we can use `stringifyJson()` instead, or maybe we can use it inside `stringifyJson()` to make it reusable.

BTW I would rename `stringifyJson()` to something that denotes it is used to make SF happy, otherwise it looks redundant until you see the implementation...